tests/uefistored/test_secure_boot.py

```diff
@@ -400,7 +402,9 @@ def test_pool_certs_present_and_disk_certs_different(self, uefi_vm):
         vm.host.pool.install_custom_uefi_certs([pool_auths[key] for key in ['PK', 'KEK', 'db', 'dbx']])
         logging.info("Installing different certs to hosts disks")
         self.install_certs_to_disks(vm.host.pool, disk_auths, ['PK', 'KEK', 'db', 'dbx'])
-        # start a VM so that certs may be synced to disk if appropriate
+        # Make sure certs are synced to disk
+        if vm.host.xcp_version > '8.2.1':
```

Comment:
Testing for 8.2.1 means that whenever we release 8.2.2 it will be expected that it has the new behaviour. This is not guaranteed. I'd test for > 8.2.99 or >= 8.3

Reply:
True